### PR TITLE
upcoming: [M3-7693] - Rename `Akamai Global Load Balancer` to `Akamai Cloud Load Balancer`

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Goes all in on the `Akamai Global Load Balancer` to `Akamai Cloud Load Balancer` rename ✏️
- Yes, I even created new feature flags 🚩 and deprecated the old ones. This probably is very extra of me, but I figured now would just be a good time to get it over with

> [!important]
> The only thing that shouldn't change as part of this PR is API endpoint URLs and Docs links URLs to prevent breaking dev for internal testers. We will do a separate PR for that and coordinate with the API team

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-01-26 at 12 07 09 PM | Screenshot 2024-01-26 at 12 06 48 PM |

### Prerequisites
- You can test with the MSW **or** use `dev-test-aglb` with the dev API

### Verification steps 
- Verify you see  to `Akamai Cloud Load Balancer` instead of `Akamai Global Load Balancer` in the UI
- Look at the diff and check my find-and-replace work

## As an Author I have considered 🤔

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by hkhalil-akamai — APPROVED** [review]
☁️

**Review by jaalah-akamai — APPROVED** [review]
☁️ 🚀

**Comment by cpathipa** [comment]
@bnussman couple files are conflicting with dev branch.

File: packages/manager/src/mocks/serverHandlers.ts
Comment (cpathipa) [inline packages/manager/src/mocks/serverHandlers.ts:1]: Should we consider renaming line [316](https://github.com/linode/manager/blob/17b9b750ae67cd27d83906396dd88de6d5f6fc7a/packages/manager/src/mocks/serverHandlers.ts#L316)

**Review by cpathipa — CHANGES_REQUESTED** [review]
Should we also consider making the folder name aglb to aclb in this PR ? 

```
src/factories/aglb -->  src/factories/aclb
src/queries/aglb --> src/queries/aclb
api-v4/src/aglb --> api-v4/src/aclb 

```

**Comment by bnussman-akamai** [comment]
Going to save those changes until we switch the API endpoints over from `aglb` to `aclb` @cpathipa

**Pull request merged**